helm/openidm/templates/idm.yaml

```diff
@@ -59,10 +59,11 @@ spec:
               fieldPath: metadata.namespace
         ports:
         - containerPort: 8080
+        - containerPort: 8444
         livenessProbe:
           httpGet:
             path: /openidm/info/ping
-            port: {{ .Values.service.internalPort }}
+            port: 8080
```

Comment:
@wstrange  - is this change okay with you? I made it because I don't want to worry about the k8s client not being able to call the mutual ssl port.